WindowsServerDocs/identity/ad-fs/deployment/how-to-connect-fed-azure-adfs.md

```diff
@@ -208,6 +208,9 @@ In order to effectively balance the traffic, the ILB should be configured with l
 Using your internal DNS server, create an A record for the ILB. The A record should be for the federation service with the IP address pointing to the IP address of the ILB. For example, if the ILB IP address is 10.3.0.8 and the federation service installed is fs.contoso.com, then create an A record for fs.contoso.com pointing to 10.3.0.8.
 This will ensure that all data trasmitted to fs.contoso.com end up at the ILB and are appropriately routed. 
 
+> [!WARNING]
+>If you are using the WID (Windows Internal Database) for your AD FS database, this value should instead be temporarily set to point to your primary AD FS server or the Web Application Proxy will fail enrollement. After you have successfully enrolled all Web Appplication Proxy servers, change this DNS entry to point to the load balancer.
```

Comment:
Codestyle/consistency: may I ask you to add the following MarkDown indent marker compatibility spacing?
```suggestion
> If you are using the WID (Windows Internal Database) for your AD FS database, this value should instead be temporarily set to point to your primary AD FS server or the Web Application Proxy will fail enrollement. After you have successfully enrolled all Web Appplication Proxy servers, change this DNS entry to point to the load balancer.
```